_posts/2019-11-18-experience-agile-2019-part-2.md

```diff
@@ -0,0 +1,221 @@
+
+- Bring focus, give clarity on what needs to be done, a certain direction, with measurable goals.
+
+<img alt="leadership-focus" src="{{ '/img/2019-11-18-experience-agile-2019-part-2/leadership-focus.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 750px;">
+
+- Break silos, make teams work together.
+
+<img alt="leadership-break-silos" src="{{ '/img/2019-11-18-experience-agile-2019-part-2/leadership-silos.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 750px;">
+
+- Be accessible, be open for questions, criticism.
+
+So leadership is very important and needs to inspire the various teams. Spotify is a big name and is used as an example for many companies. 
+But Leonoor said:
+> I hate to break it to you, but you are NOT Spotify
+
+While Spotify can be used for ideas, you probably have different dependencies, stakeholders, situational problems and so on.  
+It's thus very important to come up with a plan to scale.  
+How are you going to grow and handle problems?  
+Transparency is essential, but gets more difficult when you scale.
+
+<img alt="leadership-transparency" src="{{ '/img/2019-11-18-experience-agile-2019-part-2/leadership-transparency.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 750px;">
+
+While growing, how are you going to handle the measurement of progress?  
+The most important measurement of progress is… **a working product**. 
+
+So stop asking for progress reports! Rather, create an environment for autonomy and alignment.  
+Give the teams autonomy to create the solution, while still providing the vision and alignment. 
+Tell them what to do, but let them figure out the solution. 
+
+The image below describes it perfectly.
+
+<img alt="leadership-spotify-culture" src="{{ '/img/2019-11-18-experience-agile-2019-part-2/leadership-spotify-culture.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 750px;">
+
+The agile transformation is a challenge, but there's hopeful news.  
+It's a long journey, but it is possible to achieve.
+
+> Every change, no matter how big or small, starts with the same word... YOU!
+
+## Building a Customer Value Engine for a more Successful Company, by Mario Moreira - written by Astrid Legrand
+
+### Intro
+**Building a Customer Value Engine for a more Successful Company** aims to give enterprise leaders tools and advice to maximize customer value with a high level of agility in their companies. 
+ 
+Some methods explained during this workshop can also be easily used at a project level like, for instance, the five Rs model that helps to visualize the enterprise idea pipeline. 
+
+This tool allows to visualize ideas and treat them from the moment the idea is evoked until the moment it is realized.  
+This process consists of five stages, Record – Reveal – Refine – Realize and Release, which corresponds to a **break down** of the idea into a piece of work.
```

Comment:
Use '**The** Process' instead of '**This** process' to avoid using 'This' too often